server/s3_iem.h

```diff
@@ -56,19 +56,6 @@ extern S3Option* g_option_instance;
 #define s3_iem_fatal(loglevel, event_code, event_desc, json_fmt, ...) \
   s3_iem_(loglevel, LOG_ERROR, event_code, event_desc, json_fmt, ##__VA_ARGS__)
 
-// Note: Logs IEM message into syslog only.
-// Use this macro to send addtional information to CSM
-#define s3_iem_syslog(loglevel, event_code, event_description, ...)  \
-  do {                                                               \
-    if (loglevel == LOG_INFO) {                                      \
-      s3_syslog(loglevel, "IEC:IS" event_code ":" event_description, \
-                ##__VA_ARGS__);                                      \
-    } else {                                                         \
-      s3_syslog(LOG_ERR, "IEC:ES" event_code ":" event_description,  \
-                ##__VA_ARGS__);                                      \
-    }                                                                \
-  } while (0)
-
```

Comment:
is it intended that we remove this macro? not a misprint?

Reply:
this macro were used nowhere, its functionality fully duplicated in the neighbor macro which is used, no additional comments on why it should be there. so I deleted it